Scripts/Runtime/Entities/MemsetBufferJob.cs

```diff
@@ -13,6 +16,36 @@ namespace Anvil.Unity.DOTS.Entities
     [BurstCompile]
     public struct MemsetBufferJob<T> : IJobParallelForBatch where T : struct, IBufferElementData
     {
+        // The minimum number of elements to process in a single batch.
+        // Attempts to keep batch sizes large enough to prevent multiple cores from copying the same
+        // data into cache. This job's operation is so simple the copy into cache is probably the most expensive part.
+        //
+        // Ex: We shouldn't split a set of 10 across 5 threads.
+        //
+        // Re CacheSize: If generalizing to a 64kb cache size is good enough for Unity it's good 
+        // enough for us.
+        private static readonly float MIN_BATCH_SIZE_PER_THREAD = math.max(1f, JobsUtility.CacheLineSize * 1024 / (float)UnsafeUtility.SizeOf<T>());
+
+        /// <summary>
+        /// Calculate an ideal batch size per thread.
+        /// Aim to spread work across as many threads as possible while satisfying (in order of importance):
+        ///  - Aligning batch size to cache line size
```

Comment:
Based on comment above, might want to revisit this point, if we're trying to align the batches to the `Chunk` or `L1 Cache` size rather than line size.

Reply:
Sticking with chunk size for now. Seems to be the best estimate until we feel the need to actually fetch the CPU's cache size.

Reply:
Fixed comment
```suggestion
        ///  - Aligning batch size to chunk size
```